chart/values.yaml

```diff
@@ -386,7 +386,7 @@ etcd:
     # Name of etcd's localpv hostpath storage class.
     name: "mayastor-etcd-localpv"
     # -- Host path where local etcd data is stored in.
-    basePath: "/var/local/localpv-hostpath/{{ .Release.Name }}/etcd"
+    basePath: "/var/local/{{ .Release.Name }}/localpv-hostpath/etcd"
```

Comment:
Presumably this doesn't have any issues for upgrades as the PV's were already created anyway, but new PV's will have this new path?
@niladrih we probably need to ensure this gets updated?

Reply:
Hmmm, so this should not get updated at upgrade-time, correct?

Reply:
Yes, this can potentially patch the io-engine ds with the new path but we don't want that.

Reply:
Right. But what about the mayastor-io-engine ds, if that gets patched with new path would it be a problem ?

Reply:
As things are today no I don't think so.

Reply:
Alright then, then I'm going with no-change because both no-change and change seem harmless.